docs/technologies.md

```diff
@@ -83,6 +87,51 @@ google_cloud_options.staging_location = f"gs://{staging_location}"
 google_cloud_options.temp_location = f"gs://{temp_location}"
 ```
 
+The second important part of the `wordcount` example is the `workflow.py` module:
+
+```python
+import logging
+
+import apache_beam as beam
+import bigflow
+from apache_beam.io import WriteToText
+from bigflow.dataflow import BeamJob
+
+from .pipeline import dataflow_pipeline_options, workflow_config
+from .processing import count_words
+
+
+logger = logging.getLogger(__name__)
+
+
+def wordcount_driver(pipeline: beam.Pipeline, context: bigflow.JobContext, driver_arguments: dict):
+    logger.info(f'Running wordcount at {context.runtime_str}')
+    count_words(pipeline, WriteToText("gs://{}/beam_wordcount".format(driver_arguments['temp_location'])))
+
+
+wordcount_workflow = bigflow.Workflow(
+    workflow_id="wordcount",
+    log_config={
+        'gcp_project_id': workflow_config['gcp_project_id'],
+        'log_level': 'INFO',
+    },
+    definition=[BeamJob(
+        id='wordcount_job',
+        driver_callable=wordcount_driver,
```

Comment:
Why driver_callable? Why not job_callable?

Reply:
To make it a bit similar to the dataproc job.

Reply:
What we will gain thanks to this similarity?

Reply:
consistent experience - if you know what is the driver in a dataproc job, then you might easier figure out what is a driver in a dataflow job

Reply:
I am not sure if this is a good way. We introduce a new concept that is not commonly connected with a dataflow.

Reply:
well, if you call this parameter `job_callable` it's confusing than helpful - job inside a job? should it implement the `execute` method? etc.
@anjensan what do you think?

Reply:
#113